### ISSUE TITLE
[Quest] The Exorcism of Colonel Jules

### ISSUE THREAD

**Issue by Evilanemesis** [Jinnaix/nemesis-bug-reporting#35]
When you start the event, Darkness Released spawns. 

Actual problem: 10+ of them spawns at the same time. The time between each releases is way too fast. Check video below. 
Even a Full T3 rogue level 61 cant even solo it.

https://wowwiki.fandom.com/wiki/Quest:The_Exorcism_of_Colonel_Jules

Here is how it's supposed to be; 
https://www.youtube.com/watch?v=zHjh0i5vXRQ

**Comment by Nemiis** [comment]
And i could even restart the ritual in the middle of the ritual by talking to the dranei.

**Comment by Vallox-Nemesis** [comment]
Yeah I remember that behaviour from retail. It's meant to be pretty easy with only a few spawning per wave. Currently it's just an endless stream until the end of the quest. While I was able to do it without much issue on a paladin (in basic greens from vanilla + couple quest rewards and upgrades I found, so not overpowered gear or anything), classes without self-healing and decent AoE won't really be able to complete this without help in its current form, which will lead to frustration because it's not meant to be this difficult. Toning down the spawn frequency, while not the most blizzlike implementation, would be sufficient to "fix" this, if you're after a temporary quick fix. Otherwise, you'll have to watch that video to adjust the exact spawns.

**Comment by Jinnaix** [comment]
@sveN295 Controlled [here](https://github.com/Jinnaix/Nemesis/blob/335C/src/server/scripts/Outland/zone_hellfire_peninsula.cpp#L599)
